src/ResourceManager/Network/Commands.Network/help/Add-AzureRmVirtualNetworkGatewayIpConfig.md

```diff
@@ -31,6 +31,59 @@ Add-AzureRmVirtualNetworkGatewayIpConfig -VirtualNetworkGateway <PSVirtualNetwor
 The **Add-AzureRmVirtualNetworkGatewayIpConfig** cmdlet adds an IP configuration to a virtual network gateway.
 
 ## EXAMPLES
+### Example 1
+```
+Add-AzureRmVirtualNetworkGatewayIpConfig -VirtualNetworkGateway $gw -Name GWIPConfig2 -Subnet $subnet -PublicIpAddress $gwpip2
+
+
+Name                   : VNet7GW
+ResourceGroupName      : VPNGatewayV3
+Location               : eastus
+Id                     : /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/VPNGatewayV3/providers/Microsoft.Network/virtualNetworkGateways/VNet7GW
+Etag                   : W/"d27a784f-3c3f-4150-863d-764649b6e8e7"
+ResourceGuid           : 3c2478a7-d5d4-4e80-8532-7ea2ad577598
```

Comment:
@anzaman Is it okay to keep the Etag and ResourceGuid values?

Reply:
@maddieclayton Yes, it's fine.